specification/flag-evaluation/hooks.md

```diff
@@ -109,18 +109,24 @@ client.getValue('my-flag', 'defaultValue', new Hook3());
 > - before: API, Client, Invocation
 > - after: Invocation, Client, API
 > - error (if applicable): Invocation, Client, API
-> - finally: Invocation, Client, API> If an error occurs in the `finally` hook, it **MUST NOT** trigger the `error` hook.
+> - finally: Invocation, Client, API
 
 ##### Requirement 4.3
 
-> If an error occurs in the `before` or `after` hooks, the `error` hooks **MUST** be invoked.
+> If an error occurs in the `finally` hook, it **MUST NOT** trigger the `error` hook.
+
+In practice, this means that errors that occur in the finally hook will bubble up.
```

Comment:
This makes sense to me, maybe we should add something similar about errors in error hooks.

Reply:
Interestingly, in 4.6, it says 

> If an error is encountered in the error stage, it **MUST NOT** be returned to the user.

The big question is: How bullet proof do we expect hook authors to write their code?

I think I'll need to look at other implementations (OTel, maybe) to figure out what they've learned.